amundsen_application/static/css/_icons.scss

```diff
@@ -29,163 +73,118 @@ img.icon {
   }
 
   &.icon-alert {
-    -webkit-mask-image: url('/static/images/icons/Alert-Triangle.svg');
-    mask-image: url('/static/images/icons/Alert-Triangle.svg');
+    -webkit-mask-image: url("/static/images/icons/Alert-Triangle.svg");
```

Comment:
Note that we never made a strict lint choice for single vs double quotation marks, though I slightly favor double quotes if required to make a choice right at this moment. If you want to intentionally go this route, it would be great to have one PR that makes the changes everywhere to make diffs for future work easier to read through.

Reply:
Makes sense!
I think this was my prettier changing things, but I thought it was following the project settings, so I was wrong there.

I am open to undo the change or work on setting the project configuration to enforce these with prettier or ESLint!

Reply:
I will merge for now, but make a note about syncing about this later, thanks!